### PR TITLE
feat: add setRow API

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* added `setRow` API.
* fixed cypress `getCellByIdx` command.
* fixed bug that sorting state is not maintained on calling appendRow API.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by js87zz** [comment]
타입추출 방법 research 필요

File: packages/toast-ui.grid/index.d.ts [hunk 1]
```diff
@@ -533,6 +533,33 @@ declare namespace tuiGrid {
     tabMode?: TabMode;
   }
 
+  type IFilterOptionType = 'text' | 'number' | 'date' | 'select';
```
Comment (jung-han) [inline packages/toast-ui.grid/index.d.ts]: 이럴수가..많네요....

File: packages/toast-ui.grid/src/grid.tsx [hunk 2]
```diff
@@ -1437,7 +1437,7 @@ export default class Grid {
   }
 
   /**
-   * set column filter
+   * Sets the option of column filter.
```
Comment (jung-han) [inline packages/toast-ui.grid/src/grid.tsx]: sets 보다 set이 맞지 않을까요?

Comment (js87zz) [reply packages/toast-ui.grid/src/grid.tsx]: 기존에 다 단수형으로 주석이 달려있어서 맞췄는데.. 지금보니 꼭 그렇지도 않네요. 전부 `s`는 제거하는걸로 변경할게요

File: packages/toast-ui.grid/src/dispatch/data.ts [hunk 3]
```diff
@@ -837,3 +815,44 @@ export function updateRowNumber({ data }: Store, startIndex: number) {
...
+
+  if (prevRow && isRowSpanEnabled(sortState)) {
+    updateRowSpanWhenAppend(rawData, prevRow, false);
+  }
+
+  getDataManager(id).push('UPDATE', rawRow);
+  notify(data, 'rawData');
+  notify(data, 'viewData');
+  notify(data, 'filteredRawData');
+  notify(data, 'filteredViewData');
```
Comment (dongwoo-kim) [inline packages/toast-ui.grid/src/dispatch/data.ts]: 그러고보니 동일한 객체에 대해서는 `notify`가 여러 프라퍼티를 동시에 받아서 한 번에 처리할 수 있으면 좋겠네요.

Comment (js87zz) [reply packages/toast-ui.grid/src/dispatch/data.ts]: 넵 array wrapping 작업할때 같이 반영하면 되겠네요~ㅋ

**Pull request merged**